pr_review_time.rb

```diff
@@ -1,11 +1,12 @@
 require 'octokit'
-require 'time'
 require 'descriptive_statistics'
 
+
 ONE_HOUR = 3600
 WORK_HOURS_PER_DAY = 8
 HOURS_IN_A_DAY = 24
-REPO = ARGV[0]
+WEEKEND = ONE_HOUR * HOURS_IN_A_DAY * 2
+REPO = 'bbc/mozart-controller'
```

Comment:
Can we keep the ability to specify the repo by the 1st command line argument? :)

Reply:
oh yea sure forgot to change that back :)

Reply:
done it